packages/core/docs/guide/composables.md

```diff
@@ -348,3 +348,40 @@ watch(error, (error, prevError) => {
 In this example, we are using `useUiNotification` - a composable that handles notifications state. You can read more about it in the API reference.
 
 [//]: # 'TODO: This should be added to API reference'
+
```

Comment:
missing information about repetitive notification msgs from assets dir, it's worth mentioning that they exist